packages/util/src/orderBook/index.ts

```diff
@@ -0,0 +1,209 @@
+import type { Api } from '../api';
+
+import { map } from 'rxjs';
+import { FPNumber } from '@sora-substrate/math';
+import { HistoryItem } from '../BaseApi';
+import { LimitOrder, Side, Value } from './types';
+
+import type { Observable } from '@polkadot/types/types';
+
+export class OrderBookModule<T> {
+  constructor(private readonly root: Api<T>) {}
+
+  public orderBooks = {};
+  public userOrderBooks: Array<string> = [];
+
+  /**
+   * Get order books and set to public orderBooks object
+   *
+   */
+  public async getOrderBooks(): Promise<void> {
+    const toKey = (address) => address.code.toString();
+
+    const entries = await this.root.api.query.orderBook.orderBooks.entries();
+
+    const orderBooks = entries.reduce((buffer, [key, value]) => {
+      const orderBook = key.args[0].toJSON();
+      const meta = value.unwrap();
```

Comment:
Error: unwrapping of none value

Please use unwrap or default with the default value you set (maybe null).